app/templating/schema_context.py

```diff
@@ -1,6 +1,7 @@
-from app import settings
 from app.utilities.date_utils import to_date
 
+EQ_MAX_NUM_REPEATS = 25
```

Comment:
We could revert some of the changes from my latest commit and put the maximum number of repeats value back into`settings.py`? I made this change after discussing with @jonnyshaw89  and @adoublebarrel, since we never override this using environment variables. Whilst using config from flask is the ideal, I think it's more trouble than it's worth for this particular refactor, might as well just reference the constant directly (otherwise we have to pass the value down multiple times from to get it to where it needs to be used. So as long as everyone's happy with referencing settings file, then I'll take that approach?